service/src/main/java/tds/assessment/services/impl/AssessmentServiceImpl.java

```diff
@@ -40,8 +42,9 @@ public AssessmentServiceImpl(AssessmentQueryRepository assessmentQueryRepository
     }
 
     @Override
+    @Cacheable(CacheType.LONG_TERM)
     public Optional<Assessment> findAssessment(final String clientName, final String assessmentKey) {
-        Optional<Assessment> maybeAssessment = assessmentQueryRepository.findAssessmentByKey(assessmentKey, clientName);
+        Optional<Assessment> maybeAssessment = assessmentQueryRepository.findAssessmentByKey(clientName, assessmentKey);
```

Comment:
Noticed this bug while I was writing my integration test - parameters were switched up here.